docs/build.md

```diff
@@ -72,14 +72,14 @@ LCE converter's PIP package inside the `tensorflow:custom-op-ubuntu16`
 container. This script generates the Bazel configuration file `.bazelrc`
 in the LCE root directory.
 
-## Build LCE Runtime ##
+## Build LCE Runtime
 
 LCE runtime has a diverse platform support, covering
-[Android](./quickstart_android.md) and [ARM-based boards](./build_arm.md)
+[Android](/compute-engine/quickstart_android) and [ARM-based boards](/compute-engine/build_arm)
```

Comment:
no `https://docs.larq.dev` prefix here?

Reply:
Since the docs will most likely be accessed through the website we can omit it since it will automatically link to the right page